doc/man/julia.1

```diff
@@ -63,6 +63,12 @@ Display version information
 -h, --help
 Print help message
 
+.TP
+--project=<dir>/@.
+Activate the package environment in the given directory. The @. option will
```

Comment:
Maybe "Set dir as the home project/environment" to differentiate this from actually activating with `Pkg.activate`?

Reply:
That's a better idea, thanks!